content/spin/v2/cli-reference.md

```diff
@@ -4169,6 +4857,34 @@ Options:
 
 {{ blockEnd }}
 
+**Note:** For additional information, please see the [Managing Templates](/spin/managing-templates) and/or [Creating Templates](/spin/template-authoring) sections of the documentation.
```

Comment:
We seem to have multiple copies of this _outside_ the version tab blocks.  Not sure what this will look like but I'm betting it's not what we intend...!